assets/models/LogEntry.ts

```diff
@@ -51,6 +51,10 @@ export class SimpleLogEntry extends LogEntry<string> {
   getComponent(): Component {
     return SimpleLogItem;
   }
+
+  copyLogMessageToClipBoard(): void {
```

Comment:
This doesn't look like good logic to be part of LogEntry.ts. Models should hold data. Nothing more. Instead use the utils to do something like `function(data){...}`. Also you should look at https://vueuse.org/core/useClipboard/. It is a better solution and has a `copied` property that can be used to show a tooltip.

Reply:
yeah, I was doubtful when I was adding this. Thanks for the comment! I have isolated it to a separate component.